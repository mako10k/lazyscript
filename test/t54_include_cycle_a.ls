{- #include "test/t54_include_cycle_b.ls" -}
()
