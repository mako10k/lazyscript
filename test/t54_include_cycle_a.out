In file included from <ROOT>/test/t54_include_cycle_a.ls:1.45-89: 
In file included from <ROOT>/test/t54_include_cycle_a.ls:1.1-45: 
<ROOT>/test/t54_include_cycle_a.ls:1.89-133: error: include cycle detected for 'test/t54_include_cycle_b.ls'
